.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,5 +17,7 @@ https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requ
 | Deprecations? | yes / no
 | Fixed ticket? | Fixes #{issue number here}.
 | How to test?  | Please indicate how to best verify that this PR is correct.
+| What are the parts of PrestaShop modified by this PR?  | Please indicate what parts of the software we need to verify to check everything is alright
```

Comment:
Duplicate of https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/pull-requests/#type--category ?

![image](https://user-images.githubusercontent.com/16455155/102497526-0724ba00-4079-11eb-9c7f-c99160eab534.png)

Reply:
Not exactly, the category is less precise This new field would be used to help QA team it's additional info to the `How to test?`

However the fact that you're asking the question means that it's not clear at all and that contributors won't understand its use 😅

Reply:
Sorry but I'm lost !

This PR suggest : `What are the parts of PrestaShop modified by this PR?`
The current doc : `The category is the main part of the project affected by your changes. Choose the code that most closely describes your change:`

What is the diff ?

Reply:
@PululuK the category is actually used to build the Changelog and the Core Weekly. But "Back office" "or Web service" is very broad, it cannot help the QA.

The category aims to organize issues together while this new row aims to help QA know what to verify.